dbms/src/Storages/MergeTree/MergeTreeIndexFullText.cpp

```diff
@@ -606,8 +606,60 @@ bool SplitTokenExtractor::next(const char * data, size_t len, size_t * pos, size
 {
     *token_start = *pos;
     *token_len = 0;
+
     while (*pos < len)
     {
+#if __SSE2__
```

Comment:
#if defined(__SSE2__)

Reply:
Yep, fixed that and some other run-time issues too.